### ISSUE TITLE
feat: Add Tap Targets for first time contributors

### ISSUE THREAD

**Issue by Bhavnaharitsa** [openMF/mobile-wallet#779]
**Summary:**

We can introduce tap targets for first time contributors as there are many views in app right now.

**Comment by devansh-299** [comment]
> **Summary:**
> 
> We can introduce tap targets for first time contributors as there are many views in app right now.

@Bhavnaharitsa do you mean ShowCaseView ?

**Comment by Bhavnaharitsa** [comment]
@devansh-299 No, read about tap-targets. They come as highlights in some apps guiding the users about the features of apps. :)

**Comment by devansh-299** [comment]
> @devansh-299 No, read about tap-targets. They come as highlights in some apps guiding the users about the features of apps. :)

They seem same to me :sweat_smile:

**Comment by Bhavnaharitsa** [comment]
> > @devansh-299 No, read about tap-targets. They come as highlights in some apps guiding the users about the features of apps. :)
> 
> They seem same to me

Then, I got to read about the difference😅
